frontend/components/form/PasswordChange.vue

```diff
@@ -0,0 +1,350 @@
+                            />
+                        </div>
+                        <input
+                            id="passwordMobile"
+                            v-model="password"
+                            name="password"
+                            type="password"
+                            class="focus-ring-1 mb-3 mr-10 w-full rounded-md border-2 border-natural-400 bg-natural-100 py-1 pl-3 text-text placeholder:text-text hover:border-calypso-400 focus:outline-none dark:border-natural-700 dark:bg-natural-800 dark:text-natural-50 dark:hover:border-calypso-400"
+                        />
+                        <div
+                            class="mb-2 mr-10 flex w-full items-start text-sm text-text dark:text-natural-50"
+                        >
+                            <T
+                                key-name="dashboard.user.settings.enter.current.password"
+                            />
+                        </div>
+                        <input
+                            id="newPasswordMobile"
+                            v-model="newPassword"
+                            name="password"
+                            type="password"
+                            class="focus-ring-1 mr-10 w-full rounded-md border-2 border-natural-400 bg-natural-100 py-1 pl-3 text-text placeholder:text-text hover:border-calypso-400 focus:outline-none dark:border-natural-700 dark:bg-natural-800 dark:text-natural-50 dark:hover:border-calypso-400"
+                        />
+                        <span
+                            class="mr-10 flex w-full justify-start text-sm text-mahagony-600"
+                            >{{ errors.newPassword }}</span
+                        >
+                        <div
+                            class="mb-2 mr-10 mt-3 flex w-full items-start text-sm text-text dark:text-natural-50"
+                        >
+                            <T
+                                key-name="dashboard.user.settings.confirm.new.password"
+                            />
+                        </div>
+                        <input
+                            id="newPasswordConfirmationMobile"
+                            v-model="newPasswordConfirmation"
+                            name="password"
+                            type="password"
+                            class="focus-ring-1 mr-10 w-full rounded-md border-2 border-natural-400 bg-natural-100 py-1 pl-3 text-text placeholder:text-text hover:border-calypso-400 focus:outline-none dark:border-natural-700 dark:bg-natural-800 dark:text-natural-50 dark:hover:border-calypso-400"
+                            @keyup.enter="onSubmit"
+                        />
+                        <span
+                            class="mr-10 flex w-full justify-start text-sm text-mahagony-600"
+                            >{{ errors.newPasswordConfirmation }}</span
+                        >
+                    </div>
+                </div>
```

Comment:
Please center the button to the input fields 
![image](https://github.com/user-attachments/assets/80b675c8-13bb-4489-8b8d-3cb495e81a46)